aimmo-game-creator/worker_manager.py

```diff
@@ -268,8 +277,42 @@ def _create_game_service(self, id, _config):
         )
         service.create()
 
+    def _add_path_to_ingress(self, game_id):
+        backend = kubernetes.client.V1beta1IngressBackend("game-%s" % game_id, 80)
+        path = kubernetes.client.V1beta1HTTPIngressPath(backend, "/game-%s" % game_id)
+
+        patch = [
+            {
+                'op': 'add',
+                'path': '/spec/rules/0/http/paths/-',
+                'value': path
+            }
+        ]
+
+        self._api_instance.patch_namespaced_ingress('aimmo-ingress', 'default', patch)
+
+    def _remove_path_from_ingress(self, game_id):
+        backend = kubernetes.client.V1beta1IngressBackend('game-%s' % game_id, 80)
+        path = kubernetes.client.V1beta1HTTPIngressPath(backend, '/game-%s' % game_id)
+        ingress = self._api_instance.list_namespaced_ingress("default").items[0]
+        paths = ingress.spec.rules[0].http.paths
```

Comment:
is it safe to directly get [0] of things here?

Reply:
In fact, no. The next level down from rules will be each host - and order isn't static on the ingress yaml files it seems :/

BUT..... I guess we can cover this by the tests we will write asap.